test-otel/AsyncProfilerSpanProcessor.java

```diff
@@ -0,0 +1,33 @@
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.trace.ReadWriteSpan;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import one.profiler.AsyncProfiler;
+
+public class AsyncProfilerSpanProcessor implements SpanProcessor {
```

Comment:
I'd say, let's move everything under `test-otel` to the other repository. Eventually we're going to open a PR with this stuff there